modules/core/04-channel/v2/types/commitment.go

```diff
@@ -27,8 +27,10 @@ func CommitPacket(packet Packet) []byte {
 	appHash := sha256.Sum256(appBytes)
 	buf = append(buf, appHash[:]...)
 
+	buf = append([]byte{byte(2)}, buf...)
+
 	hash := sha256.Sum256(buf)
-	return append([]byte{byte(2)}, hash[:]...)
+	return hash[:]
```

Comment:
can we amend documentation to reflect the fact that the hash includes the prefix?